tests/python/contrib/test_hexagon/test_launcher.py

```diff
@@ -50,12 +51,12 @@ def test_add(android_serial_number, tvm_tracker_host, tvm_tracker_port):
     func.save(dso_binary_path)
 
     if not android_serial_number:
-        pytest.skip("Skip hardware test since ANDROID_SERIAL_NUMBER is not set.")
+        pytest.skip(msg="Skip hardware test since ANDROID_SERIAL_NUMBER is not set.")
 
     rpc_info = {
-      "rpc_tracker_host" : tvm_tracker_host,
-      "rpc_tracker_port" : tvm_tracker_port,
-      "rpc_server_port" : 7070,
+        "rpc_tracker_host": tvm_tracker_host,
+        "rpc_tracker_port": tvm_tracker_port,
+        "rpc_server_port": 7070,
```

Comment:
Yeah we still have that issue. I think we should try to fix it before merging your PR on adding the simulator, otherwise it would become flaky and people complain. Have you seen this issue with running on simulator?

Reply:
I haven't seen it outside of docker.  I don't know how to fix it, other than adding a sleep that will wait long enough for the socket to become available again...  Using different port numbers is guaranteed to work, but it requires paying attention when writing tests---it would be easy to make a mistake.